01week/datatype.js

```diff
@@ -0,0 +1,104 @@
+//1. Write a JavaScript program to display the current day and time.
+//Javascript provides date and time functions, utilize javascript date and time functions: getDate() returns values 1-31,get.Month() returns values 0-6,getFullYear() returns values 1900+. Make method called getDay() assign it to new Date()
+//Repeat method for time called getTime() using getHours(), getMinutes(),and getSeconds(). For minutes and seconds less that 10 run an if statement to place 0 in front of integer.
+
+var getDay = new Date();
+day = getDay.getDate();
+month = getDay.getMonth() + 1;
+year = getDay.getFullYear();
+console.log(day + "-" + month + "-" + year);
+
+var getTime = new Date(),
+  hours = getTime.getHours(),
+  minutes = getTime.getMinutes();
+seconds = getTime.getSeconds();
+if (minutes < 10) {
+  minutes = "0" + minutes;
+}
+if (seconds < 10) {
+  seconds = '0' + seconds;
+}
+console.log(hours + ":" + minutes + ":" + seconds);
+
+
+//2. Write a JavaScript program to convert a number to a string.
+//Make a var that is assigned a number then make a var to wrap the number in a string, print to log.
+var anumber = 1;
+var astring = anumber + '';
+console.log(astring);
+
+
+//3. Write a JavaScript program to convert a string to the number.
+//Use the parseInt() function which analyzes a string and returns an number.
+var a = parseInt("10");
+console.log(a);
+
+
+//4. Write a JavaScript program that takes in different datatypes and prints out whether they are a:
+//Boolean: This is conditional statement. Run an if statement to determine whether true or not
+if (10 === 2) {
```

Comment:
Wrap in a function. Also there is a javascript method that will determine the type of data, that's the method you will need to use.